components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -504,11 +504,7 @@ bool PublisherInfoDatabase::GetExcludedList(
   }
```

Comment:
let's move this function `PublisherInfoDatabase::GetExcludedList` under `PUBLISHER INFO` section

Reply:
done